test/e2e/page-objects/pages/account-list-page.ts

```diff
@@ -492,6 +494,18 @@ class AccountListPage {
     });
   }
 
+  async check_accountNotDisplayedInAccountList(
```

Comment:
we already have the method `check_accountIsNotDisplayedInAccountList` which has the same functionality

Reply:
indeed, changed